### DISCUSSION TITLE
Electribe 2 (Grey) stuck/freeze at no header when lcd screen on

### DISCUSSION THREAD

**Discussion by aldhityan** [bangcorrupt/hacktribe#307] [Q&A] [answered]
Hello all, the electribe 2 (grey) didn't start up and lcd screen on but stuck/freeze at no header. When it power on still doesn't work to power off. Also light at the bottom and blips doesn't work at all. It's an error right? Bcs I've "wrong" system.vsb didn't choose synth or sampler and it happened to software update then 'yes', it still stuck forever. Can anyone help pls, what should I do? Thanks

**Comment by bangcorrupt** [comment]
Is there writing on the screen or is it blank?  Can you post a picture?

**Reply by aldhityan** [reply]
Yes it is blank. I don't know why. Sure, here it is:
![image](https://github.com/bangcorrupt/hacktribe/assets/33146625/664b2adc-bf14-4313-b8dd-a15617945525)

Thank you

**Reply by bangcorrupt** [reply, marked as answer]
You will need to follow the [debrick guide](https://github.com/bangcorrupt/hacktribe/wiki/Debrick) to recover, see [Screen lights up](https://github.com/bangcorrupt/hacktribe/wiki/Debrick#screen-lights-up).  Also check the [RPi and FlashROM](https://github.com/bangcorrupt/hacktribe/wiki/Debrick#rpi-and-flashrom) section at the end.  If you have the right tools it may be easier, but I don't have detailed instructions.

How did you make the VSB you used?  If you email it to me I will try to work out what went wrong.  It won't help you fix this, but it could help others in future.

**Comment by aldhityan** [comment]
Hello bangcorrupt,

Thank you for your email and step by step instructions later.

I want to hacktribe make system vsb so close, but a machine is blank.

Here it is chronological order:
1. Basically download factory software original Korg: SYSTEM.VSB

2. Folder hacktribe and then in powershell:

bspatch.exe SYSTEM.VSB hacked-SYSTEM.VSB patch\hacktribe-2.patch

3. And then header:

python scripts\e2-header.py hacked-SYSTEM.VSB

4. Copy paste hacked-SYSTEM.VSB SD Card rename to SYSTEM.VSB

5. My mistake, software update in Electribe 2 is active (not invalid or
error in electribe) then 'OK'

6. That happened a header is blank :(


But I'm not sure that option/path synth or sampler in header I think, and
then I'm forgot. In the end I regret my lessons. That's it.

Thank you for patience
===========================

Best Regards,

Rinaldhi Tyan

**Reply by bangcorrupt** [reply]
> 1. Basically download factory software original Korg: SYSTEM.VSB

Did you download the synth firmware?  Hacktribe patch is for the sampler firmware.  See [this comment](https://github.com/bangcorrupt/hacktribe/discussions/66#discussioncomment-1949921).

> 2. Folder hacktribe and then in powershell:

Checking the file hash before and after patching is very important.  This ensures you have the correct file and the patch worked.  See [FAQ - What the hash is a checksum?](https://github.com/bangcorrupt/hacktribe/wiki/FAQ#what-the-hash-is-a-checksum).

> 3. And then header: 

If you used the latest version of [e2-header.py](https://github.com/bangcorrupt/hacktribe/blob/main/scripts/e2-header.py) it won't edit the file if `synth` or `sampler` is  not supplied.


Based on what you've said, it sounds like you downloaded the synth firmware, then applied the patch without checking the file hashes.  If you email me the VSB you installed, I can check.

Were you following the instructions in the [wiki](https://github.com/bangcorrupt/hacktribe/wiki/How-To), or did you use a tutorial somewhere else?
